### ISSUE TITLE
您好，可否重新提供一下.onnx预训练模型吗

### ISSUE THREAD

**Issue by workerlk** [Tokisywyy/AECNS__CAGCRN#2]
非常感谢你的开源
关于TFAG模块，有点点问题，看到在时间卷积这里，并没有在卷积前填充，感觉这里用到了未来信息？
class TFAG(nn.Module):
    """Time-Frequency Adaptive Gating module.

    This module performs multi-scale feature extraction in both time and frequency domains,
    followed by adaptive gating to selectively combine features.

    Args:
        channels (int): Number of input channels
    """
    def __init__(self, channels):
        super().__init__()

        reduced_channels = channels // 2  # Reduce channels for efficiency

        # Multi-scale temporal feature extraction
        self.temporal_scales = nn.ModuleList([
            nn.Sequential(
                nn.Conv2d(channels, reduced_channels, (5,1), padding=(2, 0), dilation=(1,1), groups=reduced_channels),
                nn.BatchNorm2d(reduced_channels),
                nn.PReLU()
            ),
            nn.Sequential(
                nn.Conv2d(channels, reduced_channels, (5,1), padding=(4,0), dilation=(2,1), groups=reduced_channels),
                nn.BatchNorm2d(reduced_channels),
                nn.PReLU()
            )
        ])
